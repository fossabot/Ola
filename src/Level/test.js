import React from 'react'
import { Level } from './'
import renderer from 'react-test-renderer'

it('Default Level', () => {
  const tree = renderer
    .create(<Level />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Level Variant Low', () => {
  const tree = renderer
    .create(<Level variant={'low'} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Level Variant Medium', () => {
  const tree = renderer
    .create(<Level variant={'medium'} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Level Variant High', () => {
  const tree = renderer
    .create(<Level variant={'high'} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
