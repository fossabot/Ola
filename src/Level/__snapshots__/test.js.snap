// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Level 1`] = `
<div
  className="ola_level is-size_medium"
>
  <span />
  <span />
  <span />
  <span />
</div>
`;

exports[`Level Variant High 1`] = `
<div
  className="ola_level is-high is-size_medium"
>
  <span />
  <span />
  <span />
  <span />
</div>
`;

exports[`Level Variant Low 1`] = `
<div
  className="ola_level is-low is-size_medium"
>
  <span />
  <span />
  <span />
  <span />
</div>
`;

exports[`Level Variant Medium 1`] = `
<div
  className="ola_level is-medium is-size_medium"
>
  <span />
  <span />
  <span />
  <span />
</div>
`;
